### DISCUSSION TITLE
General improvements suggested by ChatGPT

### DISCUSSION THREAD

**Discussion by bi6o** [official-stockfish/Stockfish#4523] [Ideas]
Wondering if any of the issues in this list is actually applicable

<img width="838" alt="Screenshot 2023-04-12 at 11 27 16" src="https://user-images.githubusercontent.com/15965988/231415660-67ee5575-d5ad-49b4-bc09-acf269b2eb90.png">

**Comment by Disservin** [comment]
No. Has been proposed before, chatgpt is trash when it comes to coming up with new logic

**Comment by bi6o** [comment]
fair enough 👍

**Comment by ginkgo20** [comment]
![image](https://github.com/official-stockfish/Stockfish/assets/132969765/c5ca979a-6da2-4804-9a15-41a443aa5adb)

**Chat GPT VS Stockfish : Who Will Win? Fight thoughts**

_For anyone who doesn’t know, Stockfish is a chess engine that claimed the title of 13-time winner of the Top Chess Engine Championship. It’s able to play very well and no matter how complex moves you throw at it, Stockfish will surprise you with its moves._

_On the other hand, ChatGPT is a chatbot created by OpenAI. It’s undoubtedly one of the smartest chatbots available on the internet right now, and it can carry out a smart conversation like human beings. And no matter what the topic is, you can talk with ChatGPT without any restrictions. What’s even more fascinating is that ChatGPT can play chess with you through algebraic chess notation._

**Here is  a good article about Chat GPT VS Stockfish**
https://allresultbd.com/chat-gpt-vs-stockfish/
